<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>829</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>图像预处理</string>
  </property>
  <widget class="QPushButton" name="btn_load_RGB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加载可见光图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next_img">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>780</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一张</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_last_img">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>780</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上一张</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="imgView">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>921</width>
     <height>701</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txt_img_number">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_all_img">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>保存全部</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load_SAR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加载SAR图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load_IR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加载红外图片</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txt_msg">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>780</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rotate">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>530</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>旋转</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_angle">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>530</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_img">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>580</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_labeling">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>200</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>标注模式</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_clipping">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>260</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>裁剪模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_label">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>80</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>保存标签</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txt_angle">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>530</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txt_class">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;类别：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_class">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>150</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gamma">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>480</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>伽马变换</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_get_train_test_data">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>780</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>生成训练集测试集</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
